OMCompiler/Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -15812,15 +15812,30 @@ public function getCMakeVersion
 protected
   Integer retVal;
   String cmakeVersionLogFile = "systemCall_cmakeVersion.log";
+  list<String> regexOut;
+  Integer numMatches;
   String cmakeVersionString;
 algorithm
-  // Regex magic to read major.minor.patch version from cmake --version
-  retVal := System.systemCall(pathToCMake + " --version 2>&1 | grep \"^.*cmake version\" | sed -e 's/^.*cmake version *//' | sed -e 's/-.*//'", cmakeVersionLogFile);
+  retVal := System.systemCall(pathToCMake + " --version", cmakeVersionLogFile);
   if 0 <> retVal then
     System.removeFile(cmakeVersionLogFile);
     Error.addInternalError("Failed to get version from " + pathToCMake, sourceInfo());
+    fail();
+  end if;
+  // Regex magic to read major.minor.patch version from cmake --version
+  // regex \d doesn't work on Linux, using [0-9] instead
```

Comment:
I'll go out on a limb here and say that whatever Linux tool this ends up using in the end does not support the PCRE regex engine (https://www.baeldung.com/linux/digit-pattern-not-supported-by-grep-regex).